src/Test/Utilities/Portable/Metadata/MetadataReaderUtils.cs

```diff
@@ -393,19 +367,38 @@ internal static void VerifyMethodBodies(ImmutableArray<byte> peImage, bool expec
                     if (rva != 0)
                     {
                         var il = peReader.GetMethodBody(rva).GetILBytes();
-                        var throwNull = new[] { (byte)ILOpCode.Ldnull, (byte)ILOpCode.Throw };
-
-                        if (expectEmptyOrThrowNull)
-                        {
-                            AssertEx.Equal(throwNull, il);
-                        }
-                        else
-                        {
-                            AssertEx.NotEqual(throwNull, il);
-                        }
+                        ilValidator(il);
+                    }
+                    else
+                    {
+                        ilValidator(null);
```

Comment:
Generally, yes, I think so.
The purpose of the refactoring is to make this helper method more generic and re-usable. If I want to validate method bodies, I would want to see empty method bodies too. 
In my specific scenario, I don't actually care though.